doc/devel/MEP/MEP28.rst

```diff
@@ -0,0 +1,62 @@
+=======================
+ MEP 28 Artist Refactor
+=======================
+
+.. contents::
+   :local:
+
+
+Branches and Pull requests
+==========================
+
+All development branches containing work on this MEP should be linked to from here.
+
+All pull requests submitted relating to this MEP should be linked to
+from here.  (A MEP does not need to be implemented in a single pull
+request if it makes sense to implement it in discrete phases).
+
+Abstract
+========
+
+During the course of MPL, the number and complexity of artists have grown, to become quite unmanageable, this PR seeks to address that by refactoring the Artists and creating a more sensible class structure.
```

Comment:
I think all of our rst files should use hard returns to keep line lengths reasonable.  I haven't checked, but I assume this has been done in the past.